lib/Common/Memory/Recycler.inl

```diff
@@ -280,7 +296,27 @@ Recycler::RealAllocFromBucket(HeapInfo* heap, size_t size)
 #endif
         )
     {
-        VerifyZeroFill(memBlock + sizeof(FreeObject), sizeCat - (2 * sizeof(FreeObject)));
+        // TODO: looks the check has been done already
+
+        bool isPageHeapAlloc = false;
```

Comment:
Isn't there a big switch indicating whether PageHeap is enabled on the recycler at all? Can we wrap this code under that so that we dont need to do any of this checking unless page heap is enabled overall?